docs/source/development/style-guide/requirement-and-issue-tracking.rst

```diff
@@ -0,0 +1,120 @@
+Level 5 requirement requirements decomposed from the level 4 requirements that better define how the SDC plans to satisfy the parent level 4 requirement, while not defining any implementation details. Example: Backup *Metadata* to AWS backup account vs. Backup *OpenSearch* to AWS backup account
+Implementation task Github issue that better defines how a certain feature will be implemented. Implementation tasks are tracked under the parent level 5 requirement as a sub-task. Each implementation task should be small enough to be handled in a single PR.
+SPIKE               Github issue created to investigate different architecture design approaches. Like implementation tasks, SPIKEs should be tracked under the applicable level 5 requirement as a sub-task.
+=================== ===================================================================================================================================================================================================================================================
+
+Requirements Overview
+^^^^^^^^^^^^^^^^^^^^^
+
+Requriements Mapping
+~~~~~~~~~~~~~~~~~~~~
+
+* Each level 4 requirement is mapped to at least one SIT.
+* Each level 4 requirement relevant to the upcoming SIT is decomposed into level 5 requirements that, together, cover the testing requirements of the SIT.
+* The level 5 requirement creation is incremental as we move through development for each SIT.
+* Each Level 5 requirement will have implementation tasks (sub-issues) that fully cover the Level 5 requirement and contain implementation specific information.
+
+.. image:: ../../_static/req-decomposition.png
+   :alt: Requirement decomposition diagram
+   :width: 1293
+   :height: 629
+
+Requirements Lifecycle
+~~~~~~~~~~~~~~~~~~~~~~
+The below image shows an example of the lifecycle of a Level 4 requirement over the course of the project:
+
+.. image:: ../../_static/req-lifecycle.png
+   :alt: Requirement lifecycle diagram
+   :width: 1631
+   :height: 621
+
+
+Overview of Managing Requirements in Github
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* All level 5 requirements will be managed as Github issues.
+* The status of the requirement issue ("Open" / "Closed") is an indication of both development and testing status, so a requirement is not "Closed" until it is implemented and has passed a SIT.
+* Requirement issues should track sub-issues that describe the implementation.
+   * Unlike the L5 requirement issues, these issues close once the code is merged.
+   * Sub-issues allow for better tracking of how SPIKES, implementation issues, and requirements are related to each other.  They also allow the use of L5 requirements as a starting point to see the design history of each feature.
+* If a re-design is proposed that impacts an already tested requirement, that requirement issue should be re-opened and any new SPIKEs or implementation tasks should be appended to the original task list.
```

Comment:
Also, we should link to or summarize whatever discussion led to us re-designing that requirement.

Reply:
Yeah, I can add more info in the text here. In the requirement issue lifecycle diagram, I said a comment needs to be added when the requirement is re-opened explaining what design change is being proposed.